samples/nhibernate/simple/NHibernate_7/Server/Program.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Threading.Tasks;
+using NHibernate.Cfg;
+using NHibernate.Mapping.ByCode;
+using NServiceBus;
+using NServiceBus.Features;
+using NServiceBus.Persistence;
+using Server;
+using Environment = NHibernate.Cfg.Environment;
+
+class Program
+{
+    static void Main()
+    {
+        AsyncMain().GetAwaiter().GetResult();
+    }
+
+    static async Task AsyncMain()
+    {
+        Console.Title = "Samples.NHibernate.Server";
+
+        #region Config
+
+        var endpointConfiguration = new EndpointConfiguration("Samples.NHibernate.Server");
+        var persistence = endpointConfiguration.UsePersistence<NHibernatePersistence>();
+
+        var nhConfiguration = new Configuration();
+        nhConfiguration.SetProperty(Environment.ConnectionProvider, "NHibernate.Connection.DriverConnectionProvider");
```

Comment:
These lines should be broken because otherwise they will not render pretty on the doco site.